### PR TITLE
add mouse x y addon

### LINKED ISSUE #535: mouse x and y in greenflag bar
**Describe the new addon**

adds the mouse x and y in the greenflag bar

**Link to the new addon**

i'm working on making it

### PR DESCRIPTION
**Pull request by JeffaloBob**
**Resolves**

Resolves #535 

**Changes**

i added a mouse x and y addon 
image
image

**Reason for changes**

because it's useful, and griffpatch suggested

**Tests**

yes i tested it by seeing if it works and it works

### CODE REVIEW

**Comment by JeffaloBob** [comment]
this one is good to go as far as i can tell

File: addons/mouse-pos/userscript.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
+export default async function ({ addon, global, console }) {
+  console.log("mouse pos enabled");
+
+  const vm = addon.tab.traps.onceValues.vm;
+
+  while (true) {
+    let bar = await addon.tab.waitForElement(".controls_controls-container_2xinB", { markAsSeen: true });
+
+    if (document.location.href.split("/")[5] == "editor") {
```
Comment (WorldLanguages) [inline addons/mouse-pos/userscript.js:9]: Use `addon.tab.editorMode`

File: addons/mouse-pos/addon.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
+{
+  "name": "Mouse position",
+  "description": "Displays your mouse position in the editor.",
```
Comment (WorldLanguages) [inline addons/mouse-pos/addon.json:3]: Maybe you should be more clear on when the x/y position appears, and where? Something like `Displays the  x and y coordinates for the mouse position next to the green flag, when on editor mode.`

**Comment by JeffaloBob** [comment]
i did the requested changes, `Displays your mouse position next to the flag in the editor.`

**Pull request merged**